modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import cv2 as cv
+import sys
+import glob
+import argparse
+
+parser = argparse.ArgumentParser(description="Run benchmark")
```

Comment:
Which benchmark? Please specify the algorithm (single object tracking) and name of the dataset.

Reply:
OK.